src/pages/woong/Detail/Comment.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function Comment(props) {
+	return (
+		<ul className='reviewLog'>
+			{props.commentArray &&
+				props.commentArray.map((comment, id) => (
+					<li key={id}>
+						<span className='nickname'>woongstaa</span>
+						<span>{comment}</span>
+					</li>
+				))}
+		</ul>
+	);
+}
+
```

Comment:
```suggestion

function Comment(props) {
  return (
    <ul className='reviewLog'>
      {props.commentArray && props.commentArray.map(
        (comment, id) => (
	  <li key={id}>
	    <span className='nickname'>woongstaa</span>
	    <span>{comment}</span>
	  </li>
	))}
    </ul>
  );
}
```

제가 알고있는 들여쓰기 convention이 스페이스 2칸이라서 코멘트 남겨봅니다.

Reply:
제 파일에선 보는게 괜찮게 나오는데 왜 이런지 모르겠네요? ㅋㅋ
수정해보겠습니다!